src/core/CellComponent/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { forwardRef, ReactNode } from "react";
+
+import { CellComponentExtraProps, StyledCellComponentData } from "./style";
+
+export interface CellComponentRawProps {
+  children?: ReactNode | React.ReactNode[];
```

Comment:
I'm not sure if `CellComponent` can take multiple ReactNodes as children! Can you confirm this with Maya?

Reply:
fixed thanks for catching